Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 3.1.3 - 2021-09-14
+
+### Fixed
+
+- Bug when two corrupted samples in a row caused all future samples to be corrupt
```

Comment:
I think it would be helpful to our users, if we also explicitly mention the case of long sessions. Since it might not be obvious, that there is a connection between two corrupted samples and long sessions. Also: Please add a period to the end of changelog entries.

```suggestion
- Two corrupted samples in a row caused all future samples to be corrupt. 
  This could especially corrupt long running measuring session.
```